src/components/AddressSearch.js

```diff
@@ -117,7 +117,13 @@ const AddressSearch = (props) => {
             sublocality: 'long_name',
             postal_code: 'long_name',
             administrative_area_level_1: 'short_name',
-            country: 'long_name',
+            country: 'short_name',
+        });
+
+        const {
+            administrative_area_level_1: stateName,
+        } = GooglePlacesUtils.getAddressComponents(addressComponents, {
+            administrative_area_level_1: 'long_name',
```

Comment:
can this not be included in the above extraction?

Reply:
Aahhhh true I see 👍  Then can we make this more clear _why_ this is needed, with:
1. A comment before this, explaining basically what you just explained
2. Maybe change`administrative_area_level_1: stateName,` to `administrative_area_level_1: longStateName,`